lib/core/managers/course_repository.dart

```diff
@@ -464,9 +461,9 @@ class CourseRepository {
   }
 
   /// Retrieve the evaluation filtered by sessions.
-  Future<Map<String, List<CourseEvaluation>>> _getCoursesEvaluations() async {
-    final Map<String, List<CourseEvaluation>> evaluations = {};
-    List<CourseEvaluation> sessionEvaluations = [];
+  Future<Map<String, List<CourseReview>>> _getCoursesReviews() async {
+    final Map<String, List<CourseReview>> evaluations = {};
+    List<CourseReview> sessionEvaluations = [];
```

Comment:
Same comment here (and a few more times in this file, but I don't feel like adding a comment for all of them if it ends up not being important 😅).
I think ignoring this minor issue would be fine, so do as you like as far as I'm concerned 👌